release/pkg/aws/ecr/ecr.go

```diff
@@ -59,6 +61,16 @@ func GetAuthToken(ecrClient *ecr.ECR) (string, error) {
 	return authToken, nil
 }
 
+func GetAuthTokenPublic(ecrClient *ecrpublic.ECRPublic) (string, error) {
```

Comment:
We already have this method in the ecrpublic package [here](https://github.com/aws/eks-anywhere/blob/main/release/pkg/aws/ecrpublic/ecrpublic.go#L50).

Reply:
O ok I'll update that